common/oauthTokenManager.go

```diff
@@ -742,8 +870,11 @@ func (credInfo *OAuthTokenInfo) GetNewTokenFromMSI(ctx context.Context) (*adal.T
 	result := &adal.Token{}
 	if len(b) > 0 {
 		b = ByteSliceExtension{ByteSlice: b}.RemoveBOM()
+		// Unmarshal will give an error for Go version >= 1.14 for a field with blank values. Arc-server endpoint API returns blank for "not_before" field.
+		// TODO: Remove fixup once Arc team fixes the issue.
+		b = fixupTokenJson(b)
```

Comment:
Just Curious: Is it a known issue? If not, have we filed the issue on Arc team?

Reply:
We have discussed this issue with the team handling managed identity auth on Arc and it will be worked upon.